src/riak_kv_qry_worker.erl

```diff
@@ -186,20 +200,27 @@ pop_next_query() ->
     self() ! pop_next_query.
 
 %%
-execute_query({query, ReceiverPid, QId, [Qry|_] = SubQueries, _},
+execute_query({query, ReceiverPid, QId, [Qry1|_] = SubQueries, _},
               #state{ run_sub_qs_fn = RunSubQs } = State) ->
     Indices = lists:seq(1, length(SubQueries)),
     ZQueries = lists:zip(Indices, SubQueries),
     %% all subqueries have the same select clause
-    ?SQL_SELECT{'SELECT' = Sel} = Qry,
-    #riak_sel_clause_v1{initial_state = InitialState} = Sel,
+    ?SQL_SELECT{'SELECT' = Sel} = Qry1,
+    #riak_sel_clause_v1{initial_state = InitialState1} = Sel,
+    InitialState2 =
```

Comment:
`riak_sel_clause_v1.initial_state` should be set in `riak_kv_qry_compiler` line 198 surely?